coldfront/core/project/templates/project/add_user_search_results.html

```diff
@@ -52,13 +54,16 @@
             <td>{{ form.first_name.value }}</td>
             <td>{{ form.last_name.value }}</td>
             <td>{{ form.email.value }}</td>
+              <td><span class="badge badge-success">{{ form.user_access_agreement.value }}</span></td>
```

Comment:
To be consistent with the home page, let's add a check mark / cross mark to the left of "Signed" / "Unsigned".

See [here](https://github.com/ucb-rit/coldfront/blob/master/coldfront/core/portal/templates/portal/authorized_home.html#L32).